.envrc

```diff
@@ -11,3 +11,6 @@ watch_file flake.lock
 } || use nix
 
 eval "$(iogo login)"
+
+export NOMAD_NAMESPACE=
+export NOMAD_TOKEN="$(vault read -field secret_id nomad/creds/admin)"
```

Comment:
Just a stupid question. When I'm doing the deployment, i'm using: 
```
export NOMAD_TOKEN="$(vault read -field secret_id nomad/creds/developer)"
```

Do i understand correctly, your change just set the default value i can override during my session?

Reply:
@dkijania yes, you could/should override it in your session (if you use direnv/nix).

Do you maybe know of a way to distinguish between a developer and devops person in vault? I think we should be smarter here and detect this somehow, just I'm not sure how.